src/statics/About.js

```diff
@@ -11,42 +11,90 @@ export default () =>
-        by building together a true sharing economy, a more free and trustworthy
-        Internet without middleman.
+        <FormattedMessage
+          id="@statics/busy_mission"
+          defaultMessage="The Busy project’s mission is to connect everyone to the next-gen decentralized social network that gives voice and value back to people, by building together a true sharing economy, a more free and trustworthy Internet without middleman"
+        />
+        .
       </p>
       <div className="About__1 hero">
         <div className="container">
-          <h1>Ensuring compensation for the creators of value</h1>
-          <p>Artists and User-generated content create billions dollars worth of
-            value absorbed by the shareholders of these intermediaries and centralized
-            services. As a non-profit and decentralized organization, Busy provides
-            the power of cryptocurrency and blockchain innovations within a platform
-            that returns much of its value to the creators ensuring fair compensation.</p>
+          <h1>
+            <FormattedMessage
+              id="@statics/ensure_compensation"
+              defaultMessage="Ensuring compensation for the creators of value"
+            />
+          </h1>
+          <p>
+            <FormattedMessage
+              id="@statics/busy_mission_2"
+              defaultMessage="Artists and User-generated content create billions dollars worth of value absorbed by the shareholders of these intermediaries and centralized services"
+            />
+            .
+            {' '}
+            <FormattedMessage
+              id="@statics/busy_mission3"
+              defaultMessage="As a non-profit and decentralized organization, Busy provides the power of cryptocurrency and blockchain innovations within a platform that returns much of its value to the creators ensuring fair compensation"
+            />
+            .
+          </p>
         </div>
       </div>
       <div className="About__2 hero">
         <div className="container">
-          <h1>Creating a true sharing economy</h1>
-          <p>Uber or Airbnb are often described as the sharing economy, a nice notion
-            that peers create and share in value. But these businesses have little to
-            do with sharing, they in fact simply aggregate as middleman through their
-            centralized, proprietary platforms and do resell by taking huge commissions.
-            Today's big disrupters are about to get disrupted ! Busy aims to create a
-            true peer-to-peer sharing economy without middleman and zero transaction
-            fee that allows people to transact safely and freely.</p>
+          <h1>
+            <FormattedMessage
+              id="@statics/sharing_economy"
+              defaultMessage="Creating a true sharing economy"
+            />
+          </h1>
+          <p>
+            <FormattedMessage
+              id="@statics/sharing_economy_2"
```

Comment:
Why separate a text into multiple translations text? This will probably change and will require more work to maintain with lot of translation isnt?